crates/interpreter/src/interpreter/input.rs

```diff
@@ -22,7 +22,13 @@ impl InputsTr for InputsImpl {
     }
 
     fn input(&self) -> &[u8] {
-        &self.input
+        match &self.input {
+            CallInput::Range(_range) => {
```

Comment:
Will need to move this to have access to shared memory, but for this PR this lgtm